### PR TITLE
Manage public elements

### PR DESCRIPTION
**Pull request by sowmyaraj92**

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/manage-public-elements.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+<img alt="Manage workspaces" src="https://assets.postman.com/postman-docs/manage-workspaces.jpg"/>
+
+ To respond to the request, click **Respond** towards the right end of the workspace. A dialog box will open up in the workspace overview tab. Upon approving, the workspace gets converted to a [public workspace](/docs/collaborating-in-postman/public-workspaces/).
+
+ <<img alt="Convert to public workspaces" src="https://assets.postman.com/postman-docs/request-visibility-public-workspace.jpg"/>>
+
+#### Public Documentation
+
+The Documentation tab displays all the collections with published documentation. The search box below the header enables you to filter by the publisher. Along with the collection name and environment name, you will be able to view date the documentation was published on, who published it, and the documentation URL. Copy the **Documentation URL** and paste it in a browser of your choice, and you will have access to the documentation.
```
Comment (jkonrath-postman) [inline src/pages/docs/collaborating-in-postman/manage-public-elements.md:40]: **Documentation** tab

File: src/pages/docs/collaborating-in-postman/manage-public-elements.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+<img alt="Public documentation" src="https://assets.postman.com/postman-docs/public-documentation.jpg"/>
+
+#### Collection JSON links
+
+The Collection JSON Links tab displays all the collections with JSON links. You can use the toggle option to enable sharing collection JSON via public links.
+
+Being the Community Manager, you will be able to view, access, or delete the collections with JSON links. In addition to the collection name, the dashboard displays more information about the date the link was updated on, who updated the link, and the JSON link.
```
Comment (jkonrath-postman) [inline src/pages/docs/collaborating-in-postman/manage-public-elements.md:48]: Change "Being" to "As"

File: src/pages/docs/collaborating-in-postman/manage-public-elements.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+If you disable the toggle option, no new links can be generated and existing links cannot be modified/edited. The previously created JSON links will still remain accessible.
```
Comment (jkonrath-postman) [inline src/pages/docs/collaborating-in-postman/manage-public-elements.md:50]: "modified/edited" to "modified or edited"

File: src/pages/docs/collaborating-in-postman/manage-public-elements.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+Hover over the **JSON link** corressponding to the collection and click **Copy link to JSON**. Paste the link in a browser of your choice and you will have access to the collection JSON.
```
Comment (jkonrath-postman) [inline src/pages/docs/collaborating-in-postman/manage-public-elements.md:52]: corressponding/corresponding

**Review by jkonrath-postman — CHANGES_REQUESTED** [review]
A few minor changes - I know this is a work-in-progress, so feel free to tag me for a re-review.

**Review by mark-anthony-postman — APPROVED** [review]
Great work putting together a shiny new page! I made some suggestions--they are all minor but might help readability a bit. Consider using them if you wish 😃

File: src/pages/docs/collaborating-in-postman/manage-public-elements.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
+---
+title: "Managing public elements"
+order: 76.1
+page_id: "manage_public_elements"
+warning: false
+contextual_links:
```
[the remark this answers is not in the document]
Comment (sowmyaraj92) [reply src/pages/docs/collaborating-in-postman/manage-public-elements.md:6]: Yes yes, I need to add them. That was the whole point of keeping contextual links so I remember to add the blogpost and page details on Monday. Thank you for the heads up! :)

**Review by gitThingsDone — APPROVED** [review]
LGTM

File: src/pages/docs/collaborating-in-postman/manage-public-elements.md [hunk 6]
```diff
@@ -55,4 +74,4 @@ Hover over the **JSON link** corresponding to the collection and click **Copy li
 
 ## Next steps
 
-For more details on how to create a public workspace, visit [Public workspaces](/docs/collaborating-in-postman/public-workspaces/).
+To learn more about workspaces, see [Creating workspaces](/docs/collaborating-in-postman/using-workspaces/creating-workspaces/) and [Managing workspaces](docs/collaborating-in-postman/using-workspaces/managing-workspaces/). For more details on how to create a public workspace, visit [Public workspaces](/docs/collaborating-in-postman/public-workspaces/).
```
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/manage-public-elements.md:77]: Add leading "/" before "docs" in the Managing workspaces link

File: src/pages/docs/collaborating-in-postman/manage-public-elements.md [hunk 7]
```diff
@@ -4,12 +4,27 @@ order: 76.1
...
+  - type: section
+    name: "Additional Resources"
+  - type: subtitle
+    name: "Related Blog Posts"
+  - type: link
+    name: "Govern Your Public API Collections More Effectively"
+    url: "https://blog.postman.com/govern-your-public-api-collections-more-effectively/"
+  - type: link
+    name: "How to Set Up Your First Public Workspace"
+    url: " https://blog.postman.com/how-to-set-up-public-workspaces/"
```
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/manage-public-elements.md:16]: Remove extra space between quotation mark and **https**

File: src/pages/docs/collaborating-in-postman/manage-public-elements.md [hunk 8]
```diff
@@ -4,12 +4,27 @@ order: 76.1
...
+    name: "Govern Your Public API Collections More Effectively"
+    url: "https://blog.postman.com/govern-your-public-api-collections-more-effectively/"
+  - type: link
+    name: "How to Set Up Your First Public Workspace"
+    url: " https://blog.postman.com/how-to-set-up-public-workspaces/" 
+  - type: section
+    name: "Next Steps"
+  - type: link
+    name: "Creating workspaces"
+    url: "docs/collaborating-in-postman/using-workspaces/creating-workspaces"
```
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/manage-public-elements.md:21]: Add leading "/" before "docs" in Creating workspaces link

**Review by mark-anthony-postman — CHANGES_REQUESTED** [review]

**Review by mark-anthony-postman — APPROVED** [review]

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**